src/Connector.sol

```diff
@@ -135,18 +135,19 @@ contract CentrifugeConnector {
         memberlist.updateMember(user, validUntil);
     }
 
-    function deposit(
```

Comment:
Okay I know I already asked for these to be renamed once, but I wonder if it wouldn't be clearer to name them `transferIn`, and `transferOut`. I worry `transfer` and `transferTo` will be confusing. What do you think?

Reply:
sure, open to rename